### ISSUE TITLE
B-Table: cannot style `background-color` of selected row

### ISSUE THREAD

**Issue by jippy89** [bootstrap-vue/bootstrap-vue#3044]
**Little Background**  
I'm just trying to style the selected row, so it can be more beautiful. However it seems that my custom style didn't get used by `b-table`, please check JSFiddle on CSS part. I can change other things, however `background-color` prop seems to stay fixed based on `select-variant` prop.

And **in fact**, the `background-color` is used by `b-table` (can be seen in Chrome Dev Tools), but the color doesn't change!

- **Operating system**: (Windows 10)
- **Browser and version**: Chrome Version 73.0.3683.103 (Official Build) (64-bit)
- **BootstrapVue**: @latest 
- [Link to JSFiddle](https://jsfiddle.net/irfandyjip89/sa957fte/)

**Comment by jippy89** [comment]
Nevermind, I solved it by setting this on my `.vue` file
```css
>>> .table-primary, >>> .table-primary>td, >>> .table-primary>th {
		background-color: transparent!important;
	}
```
Turns out even though, the specifity of `.table-primary` is crossed (on Dev Tools) it's still used by Bootstrap-Vue**(?)**, so I solved it by rewriting the `.table-primary` to transparent. Any other better solution can be mentioned tho'.

I don't know if it's a bug or what, so I'll just let this issue open. If it isn't then feel free to close it.
Thank you :)
